test/LightRelay.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
```

Comment:
why did you not use the unmodified one from tbtc?

Reply:
What do you mean?